connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -213,3 +213,13 @@ message Sample {
     int64 seed = 1;
   }
 }
+
+// Relation alias.
+message SubqueryAlias {
+  // Required. The input relation.
+  Relation input = 1;
+  // Required. The alias.
```

Comment:
how can we check if it's present or not, given the default value is empty string?

Reply:
Given our discussion on the protocol, this is a required field so we ask clients to always set it. Server side only fetch the value in this field not matter what it is (either `""`, or not)